data-source/blocks/confirmation.json

```diff
@@ -0,0 +1,18 @@
+{
+    "type": "Confirmation",
+    "id": "confirmation",
+    "title": "You\u2019re ready to submit your 2017 Census Test",
```

Comment:
Might be worth changing any of the `\u2019`/`\u2018` to actual smart quotes as jsonnet seems to have compiled it to `"You’re ready to submit your 2017 Census Test",`. So just for consistency.